src/functionalTest/resources/features/F-206 - Request Notice Of Change/common/verify-case-events/F-206_Verify_NoC_Request_Event_Data_Dil.td.json

```diff
@@ -0,0 +1,43 @@
+{
+  "_guid_": "F-206_Verify_NoC_Request_Event_Data_Dil",
+  "_extends_": "F-206_Verify_NoC_Request_Event_Data__Base",
+
+  "specs": [
+    "to get Case Events API returns a NoCRequest event in which the user ID is set to Dil's email address AND the proxied_by field set to the ID of the system user"
+  ],
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body" : {
+      "auditEvents": [
+        {
+          "id": "[[ANYTHING_PRESENT]]",
+          "description" : "[[ANYTHING_PRESENT]]",
+          "user_id": "${[scenarioContext][parentContext][testData][users][invokingUser][id]}",
+          "user_last_name": "Solicington",
+          "user_first_name": "Bill",
```

Comment:
Should this be Dil ?

Reply:
The account used for Dil is the Bill user